.github/workflows/smoke-tests.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-java@v1
         with:
-          java-version: 11
+          java-version: 17
       - uses: actions/cache@v1
         with:
           path: ~/.m2/repository
```

Comment:
@olenagerasimova is it okay that here in key is used `jdk-11-maven` instead of `jdk-17`?

Reply:
@genryxy thanks, corrected